src/kongState.js

```diff
@@ -84,8 +83,11 @@ export default async (adminApi) => {
     }));
 
     const allPlugins = await adminApi.fetchGlobalPlugins();
+    const isKongVersion1 = semVer.gte(version, '1.0.0');
     const globalPlugins = allPlugins.filter(plugin => {
-        return plugin.api_id === undefined && plugin.service_id === undefined && plugin.route_id === undefined;
+        const serviceId = isKongVersion1 ? (plugin.service || {}).id : plugin.service_id;
+        const routeId = isKongVersion1 ? (plugin.route || {}).id : plugin.route_id;
+        return serviceId === undefined && routeId === undefined;
```

Comment:
Is this expression missing `plugin.api.id`?

Reply:
yes but that's on purpose - Kalil ought to be removing it on his end and there's no sense trying to make it work with the old version (APIs are deprecated in 0.13.x iirc and not present in 0.14.x at all)